website/blog/2020-11/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Gardener v1.11 and v1.12 Released
+linkTitle: Gardener v1.11 and v1.12
+newsSubtitle: November 4, 2020
+type: blog
+publishdate: 2020-11-04
+archivedate: 2020-12-31
+authors:
+- name: Tim Usner
+  email: tim.usner@sap.com
+  avatar: https://avatars2.githubusercontent.com/u/40451181?s=460&u=4df34635cf86b924700ef4152ec8462eeaa35721&v=4
+aliases: ["/blog/2020/11/04/00"]
+---
+
+Two months after our last Gardener release update, we are happy again to present release v1.11 and v1.12 in this blog post. Control plane migration, load balancer consolidation, new security features are just a few topics we progressed with. As always, a detailed list of features, improvements, and bug fixes can be found in the [release notes](https://github.com/gardener/gardener/releases) of each release. If you are going to update from a previous Gardener version, please take your time to go through the action items in the release notes.
+
+## Noteable Changes in v1.12
+
+Release v1.12, fresh from the oven, is shipped with plenty of improvements, features and some API changes we want to pick up in the next sections. 
+
+### Functionless DNS Providers ([gardener/gardener#3036](https://github.com/gardener/gardener/pull/3036))
```

Comment:
Maybe add "Drop" in front? Otherwise at a glance it sounds like add support for that. Of course the text below is clear enough but you might want to consider people not reading the fine print :-)